src/buildblock/ProjDataInfoCylindrical.cxx

```diff
@@ -269,6 +271,7 @@ initialise_ring_diff_arrays() const
         const float ring1_plus_ring2_float =
           2*ax_pos_num/get_num_axial_poss_per_ring_inc(s_num)
           -2*m_offset[s_num]/ring_spacing + (get_scanner_ptr()->get_num_rings()-1);
+        // ORIGINTODO: ^
```

Comment:
I don't think any changes needed

Reply:
Agree, I may have briefly confused myself between ring_spacing and voxel_spacing here.